### PR TITLE
Add English version for next week's Coding4Kids event

### PR DESCRIPTION
**Pull request by realJohnDoe**
I just added an English translation for next week's Coding4Kids. The main work was done with Google Translate but I read once over it and smoothened it out where necessary.

### CODE REVIEW

**Review by carlrobert — APPROVED** [review]

File: Eyes/eyes-en.md [hunk 1]
```diff
@@ -0,0 +1,379 @@
...
+```python
+import pgzrun
+import pygame
+import math
+
+WIDTH, HEIGHT = 530, 400
+
+
+def update():
+     Pass
```
Comment (carlrobert) [inline Eyes/eyes-en.md]: pass

File: Eyes/eyes-en.md [hunk 2]
```diff
@@ -0,0 +1,379 @@
...
+
+```python
+import pgzrun
+import pygame
+import math # newline 👀
+
+WIDTH, HEIGHT = 530, 400
+
+def update():
+     Pass
```
Comment (carlrobert) [inline Eyes/eyes-en.md]: pass

File: Eyes/eyes-en.md [hunk 3]
```diff
@@ -0,0 +1,379 @@
...
+  
+```python
+import pgzrun
+import pygame
+import math
+
+WIDTH, HEIGHT = 530, 400
+
+def update():
+     Pass
```
Comment (carlrobert) [inline Eyes/eyes-en.md]: pass

File: Eyes/eyes-en.md [hunk 4]
```diff
@@ -0,0 +1,379 @@
...
+
+```python
+import pgzrun
+import pygame
+import math
+
+WIDTH, HEIGHT = 530, 400
+
+def update():
+     Pass
```
Comment (carlrobert) [inline Eyes/eyes-en.md]: pass

File: Eyes/eyes-en.md [hunk 5]
```diff
@@ -0,0 +1,379 @@
...
+```
+:pencil2: **Update the changed lines and test the code.** Do both eyes follow the mouse pointer correctly?
+>Remember to adjust the indentation in the code when adding the `draw_eye()` function.
+    
+# Tasks
+
+## 1. Evaluate your own work!
+When you answer this, imagine that *you have access to the task* &ndash; so you don't need to know the code by heart.
+
+**1A.** We have done these parts of the task. We understand them and can explain the code to Susanne or in front of the class.
```
Comment (carlrobert) [inline Eyes/eyes-en.md]: replace "Susanne" with "your mentor/teacher"

**Comment by realJohnDoe** [comment]
@carlrobert I don't have permissions to merge in this repository. Would you be so kind?

**Comment by carlrobert** [comment]
@realJohnDoe Squashed and merged. You should now have those rights as well, else ping me :)

**Pull request merged**